### PR TITLE
DEVOPS-1523 Improve Ingress Controller scheme with cron keeper

### PR DESCRIPTION
**Pull request by JustGetLucky**
# DEVOPS-1523  Improve Ingress Controller scheme with cron keeper

## What this PR does / why we need it:
The main point of improvement is reducing the count of containers in controller pods and in post-analytics pod

## Types of changes

- [x] Removed containers from post-analytics pod: `brute-detect`, `export-attacks`, `export-counters` and `sync-markers`
- [x] Removed  containers from controller pod: `sync-ip-lists` and `sync-ip-lists-source`
- [x] `exportenv` container became init container and run only once during starting pods (controller and post-analytics)
- [x] Added new `cron` container into controller and post-analytics pods, with logic from removed containers.

## How Has This Been Tested?
The changes have been tested using local minukube and audit wallarm cloud. Additional tests should be done using GKE.

## Checklist:
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: charts/ingress-nginx/templates/controller-configmap-cron.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ template "ingress-nginx.wallarmControllerCronConfig" . }}
+  namespace: {{ .Release.Namespace }}
+data:
+  crontab: |
+    # export-environment
+    0 */1 * * * timeout 10m /opt/wallarm/ruby/usr/share/wallarm-common/export-environment -l STDOUT
```
Comment (MrEcco) [inline charts/ingress-nginx/templates/controller-configmap-cron.yaml:15]: @dmitry-j-mikhin Do we need to template cron expressions and timeouts here?

Comment (dmitry-j-mikhin) [reply charts/ingress-nginx/templates/controller-configmap-cron.yaml:15]: @MrEcco why you want to merge this PR to ingress repo? I think you should create separate repo for sidecar and merge this PR to sidecar repo.

Comment (MrEcco) [reply charts/ingress-nginx/templates/controller-configmap-cron.yaml:15]: We want to use the same images for new schema then it possible. I may reduce the complexity of implementation.

Comment (JustGetLucky) [reply charts/ingress-nginx/templates/controller-configmap-cron.yaml]: @MrEcco resolved

**Review by dmitry-j-mikhin — APPROVED** [review]

**Pull request merged**